data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,86 @@
+
+## 2021 Plan for Releases
+
+Below is our current release schedule for 2021:
+
+| Electron | Chrome | Alpha Release | Beta Release | Stable Release | Stable Cycle (Weeks) |
+| -------- | ------ | ----- | ---- | ------ | -------- |
+| E13 | M91 | -          | 2021-Mar-05 | 2021-May-25 | 12 |
+| E14 | M93 | -          | 2021-May-26 | 2021-Aug-31 | 14 |
+| E15 | M94 | 2021-Jul-20 | 2021-Sep-01 | 2021-Sep-21 | 9 (includes alpha) |
+| E16 | M96 |  - | 2021-Sep-22 | 2021-Nov-16 | 8 |
+| E17 | M98 |  - | 2021-Nov-17 | 2022-Feb-01 | 11 |
+
+Adding the alpha channel extends the development time before Electron 15's launch from 3 weeks to 9 weeks - closer to our new 8 week cycle, while still meeting the requirements for Windows Store submission.
+
+To further help app developers, **for the remainder of 2021, we will also be extending our supported versions policy from the latest 3 versions to the latest 4 versions of Electron.** That means that even if you can't immediately alter your upgrade schedule, older versions of Electron will still receive security updates and fixes.
+
+## Addressing Concerns 
+
+There's a reason we're publishing this post well before this release cycle change is scheduled. We know that a faster release cycle will have a real impact on Electron apps - some of which may already find our major release cadence aggressive.
+
+We've tried to address common concerns below:
+
+#### ❓ Why even make this change? Why not keep the 12 week release cadence?
+
+To deliver the most up-to-date versions of Chromium in Electron, our schedule needs to track theirs. More information around Chromium's release cycle can be found [here](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md).
+
+Every new Chromium release contains new features, bug fixes / security fixes, and V8 improvements. We want you, as app developers, to have these changes in a timely manner, so our stable release dates will continue to match every other Chromium stable release. As an app developer, you'll have access to new Chromium and V8 features and fixes sooner than before.
+
+#### ❓ The existing 12 week release schedule already moves quickly. What steps are the team taking to make upgrading easier?
+
+One plus of more frequent releases is having _smaller_ releases. We understand that upgrading Electron's major versions can be difficult; we hope that, by releasing more frequently, we can introduce fewer major Chromium and Node changes, as well as fewer breaking changes, per release.
+
+#### ❓ Will Electron extend the number of supported versions?
+
+For the remainder of 2021, we will also be extending our supported version policy from the latest three versions to the latest four versions of Electron. When the 8 week cadence begins in 2022, we'll continue to [support the latest three major versions](https://www.electronjs.org/docs/tutorial/support#supported-versions), as well as the beta, alpha and nightly release.
```

Comment:
Follow-up: we should write a note in the linked doc.

Reply:
CONTRIBUTING change here: https://github.com/electron/electron/pull/30115
@sofianguy is changing Supported Versions, will also link that PR to merge after this blog is published.